client/js/components/actionbar/action-bar.js

```diff
@@ -1,14 +1,33 @@
 var React = require('react');
+var Link = require('react-router-component').Link;
+var classnames = require('classnames');
 var AddEvent = require('./add-event');
+var AddEmployee = require('./add-employee');
+var AllActions = require('./all-actions');
 
-var ActionBar = React.createClass({
+
+var Actionbar = React.createClass({
   render: function () {
+
+    var Action;
+    switch (this.props.hashRoute) {
+      case '#actionbar/add-event': Action = AddEvent; break;
+      case '#actionbar/add-employee': Action = AddEmployee; break;
+      default: Action = AllActions;
```

Comment:
Yea the idea is if we ultimately want to show context-based links on the sidebar or even if all the possible actions can't fit on the sidebar, I wanted a single location that could access all of the actions. It makes for a good default case as well, rather than a blank action bar.